ydb-example-urlshortener/Cargo.toml

```diff
@@ -3,17 +3,14 @@ publish = false
 name = "ydb-example-urlshortener"
 version = "0.0.0"
 edition = "2021"
-rust-version = "1.68.0"
-
+rust-version = "1.81.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-async_once = "0.2.6"
-hashers = "1.0.1"
-lazy_static = "1.4.0"
-serde = { version = "1.0", features = ["derive"] }
-tokio = { version = "1.18", features = ["macros", "rt-multi-thread", "net"] }
+hashers = "1"
+serde = { version = "1", features = ["derive"] }
```

Comment:
The dependencies set to minimum supported version. If somebody want to update the version - it is free. Anybody can update version of the serde/tokio/... crate for own lib or end binary.

Reply:
Get it!